packages/brain/src/modules/apiClients/dappnodeSignerProover.ts

```diff
@@ -0,0 +1,32 @@
+import { StandardApi } from "./index.js";
+import path from "path";
+import {
+  Network,
+  DappnodeSigningProoverPostRequest,
+} from "@stakingbrain/common";
+
+export class DappnodeSigningProover extends StandardApi {
+  private dappnodeSignEndpoint = "/newSignature";
+
+  constructor(network: Network, validatorsMonitorUrl?: string) {
+    super(
+      {
+        baseUrl: validatorsMonitorUrl || "https://dappnode-signatures.io",
+      },
+      network
+    );
+  }
+
+  public async sendProofOfAttestation(
+    proofOfAttestations: DappnodeSigningProoverPostRequest[]
+  ): Promise<void> {
+    await this.request({
```

Comment:
I'd define a timeout here to ensure we do not wait too many seconds in case our server is not answering when being sent the attestations